metadata.json

```diff
@@ -1,15 +1,12 @@
 {
-  "name": "puppetlabs-mcollective",
-  "version": "2.0.0",
-  "author": "puppetlabs",
+  "name": "puppet-mcollective",
+  "version": "2.1.0",
```

Comment:
So that we can say 'before 2.0.0 and before was puppetlabs, 2.1.0 and after was puppptcommunity'